synfig-studio/src/gui/widgets/widget_waypoint.cpp

```diff
@@ -294,5 +289,62 @@ Widget_Waypoint::get_waypoint()const
 	return waypoint;
 }
 
+void
+Widget_Waypoint::show_tcb_params(bool ok)
```

Comment:
Maybe rename ok -> show_labels?

Reply:
OK

Reply:
The function does not only show labels but also show spin buttons and set adjustment values. Moreover, the function name says show_tcb_params so I think we need a better argument name than show_labels? If "ok" was too vague what about "enable or enabled"?

Reply:
The problem with "ok" is it looks like to store success or failure

Reply:
What about changing from

`show_tcb_params(bool show_labels)`

to

`config_tcb_params(bool show_params)`

That way both func name & arg name doesn't say *show* which is quite redundant. I'm all ears to other names too :)

Reply:
Yeah, show_params is more clear